packages/cactus-test-tooling/src/main/typescript/iroha/iroha-test-ledger.ts

```diff
@@ -0,0 +1,352 @@
+    const fnTag = `IrohaTestLedger#constructor()`;
+    if (!options) {
+      throw new TypeError(`IrohaTestLedger#ctor options was falsy.`);
+    }
+    Checks.nonBlankString(options.postgresHost, `${fnTag} postgresHost`);
+    Checks.truthy(options.postgresPort, `${fnTag} postgresPort`);
+
+    this.postgresHost = options.postgresHost;
+    this.postgresPort = options.postgresPort;
+
+    this.imageVersion =
+      options.imageVersion || IROHA_TEST_LEDGER_DEFAULT_OPTIONS.imageVersion;
+    this.imageName =
+      options.imageName || IROHA_TEST_LEDGER_DEFAULT_OPTIONS.imageName;
+    this.rpcToriiPort =
+      options.rpcToriiPort || IROHA_TEST_LEDGER_DEFAULT_OPTIONS.rpcToriiPort;
+    this.envVars = options.envVars || [
+      ...IROHA_TEST_LEDGER_DEFAULT_OPTIONS.envVars,
+    ];
+
+    this.envVars.push(`IROHA_POSTGRES_HOST=${this.postgresHost}`);
+    this.envVars.push(`IROHA_POSTGRES_PORT=${this.postgresPort}`);
+
+    this.emitContainerLogs = Bools.isBooleanStrict(options.emitContainerLogs)
+      ? (options.emitContainerLogs as boolean)
+      : true;
+
+    this.validateConstructorOptions();
+    const label = "iroha-test-ledger";
+    const level = options.logLevel || "INFO";
+    this.log = LoggerProvider.getOrCreate({ level, label });
+  }
+
+  public getContainer(): Container {
+    const fnTag = "IrohaTestLedger#getContainer()";
+    if (!this.container) {
+      throw new Error(`${fnTag} container not yet started by this instance.`);
+    } else {
+      return this.container;
+    }
+  }
+
+  public get imageFqn(): string {
+    return `${this.imageName}:${this.imageVersion}`;
+  }
+
+  public async getRpcToriiPortHost(): Promise<string> {
+    const ipAddress = "127.0.0.1";
+    const hostPort: number = await this.getRpcToriiPort();
+    return `http://${ipAddress}:${hostPort}`;
+  }
+
+  /**
+   * Output is based on the standard Iroha genesis.block contents.
+   *
+   * @see https://github.com/hyperledger/iroha/blob/main/example/genesis.block
+   */
+  public getGenesisAdminAccount(): string {
+    return "admin@test";
+  }
+
+  /**
+   * Output is based on the standard Iroha admin user public key.
+   *
+   * @see https://github.com/hyperledger/iroha/blob/main/example/admin%40test.pub
+   * @see https://github.com/hyperledger/iroha/blob/main/example/genesis.block
+   */
+  public getGenesisAccountPubKey(): string {
+    return "313a07e6384776ed95447710d15e59148473ccfc052a681317a72a69f2a49910";
+  }
+
+  /**
+   * Output is based on the standard Iroha admin user private key.
+   *
+   * @see https://github.com/hyperledger/iroha/blob/main/example/admin%40test.priv
+   */
+  public getGenesisAccountPrivKey(): string {
+    return "f101537e319568c765b2cc89698325604991dca57b9716b58016b253506cab70";
+  }
+
+  public async getNodeKeyPair(): Promise<IKeyPair> {
+    const fnTag = `IrohaTestLedger#getNodeKeyPair()`;
+    if (!this.container) {
+      throw new Error(`${fnTag} this.container cannot be falsy.`);
```

Comment:
```suggestion
      throw new Error(`${fnTag} this.container cannot be false.`);
```

Reply:
I believe that 'falsy' is a common language used in the Javascript world.
[()](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)

Reply:
Ups, sorry, I didn't know, I though that it was a typo:D